### PR TITLE
524 adjust material page

### PR DESCRIPTION
**Pull request by vyacheslav1618**
#524 
Снимок экрана от 2022-12-20 18-41-51

Снимок экрана от 2022-12-16 15-04-00
Снимок экрана от 2022-12-16 14-35-56

### CODE REVIEW

**Comment by vyacheslav1618** [comment]
there are many sub tasks here, and I didn't repair a tests yet.

File: packages/frontapp/src/app/components/Search/index.tsx [hunk 1]
```diff
@@ -3,6 +3,7 @@ import styled from '@emotion/styled';
 import { colors, dimensions, fonts } from '@mimir/ui-kit';
 import { ReactComponent as SearchIcon } from '../../../assets/Navbar/Search.svg';
 import Input from '../Input';
+import { ThemeProvider } from '@emotion/react';
```
Comment (IAlexHilimovich) [inline packages/frontapp/src/app/components/Search/index.tsx:6]: Unused import

File: packages/frontapp/src/app/components/UserList/__snapshots__/SingleUser.spec.tsx.snap [hunk 2]
```diff
@@ -47,13 +47,20 @@ exports[`render SingleUserCard component renders correctly, with snapshots 1`] =
...
   flex-direction: column;
   margin-left: 1rem;
-  row-gap: 0.5rem;
 }
 
 .emotion-3>p:first-of-type {
   margin-bottom: 0.5rem;
 }
 
+@media (max-width: 48rem) {
```
Comment (IAlexHilimovich) [inline packages/frontapp/src/app/components/UserList/__snapshots__/SingleUser.spec.tsx.snap:56]: Why use rem here? Will it still work if rem value changes?

File: packages/frontapp/src/app/globalUI/Table/index.tsx [hunk 3]
```diff
@@ -70,7 +104,7 @@ const Table: FC<{ columnTitles: string[]; rows?: ReactElement[] }> = ({
   return (
     <TableContainer>
       <List>
-        <Title>
+        <Title visible={false}>
```
Comment (IAlexHilimovich) [inline packages/frontapp/src/app/globalUI/Table/index.tsx:107]: I think `visible` prop is not needed. You can make check if `Title` should be displayed, here.
Use https://www.npmjs.com/package/react-responsive to check screen-size

File: packages/frontapp/src/app/components/UserList/__snapshots__/SingleUser.spec.tsx.snap (hunk 2 above)
Comment (Chichmarin17) [reply packages/frontapp/src/app/components/UserList/__snapshots__/SingleUser.spec.tsx.snap:56]: That's an auto generated snapshot

**Review by Chichmarin17 — APPROVED** [review]

File: package.json [hunk 4]
```diff
@@ -128,6 +128,7 @@
     "lodash": "^4.17.21",
     "metro-config": "0.70.1",
     "nodemailer": "6.7.8",
+    "package.json": "^2.0.1",
```
[the remark this answers is not in the document]
Comment (vyacheslav1618) [reply package.json]: I didn't see it. Maybe IDE created it automatically with my help. I don't even imagine what it means.

Did you delete it?

[the remark this answers is not in the document]
Comment (Chichmarin17) [reply package.json]: yes

**Pull request merged**